pkg/bootstrap/docker/up.go

```diff
@@ -755,7 +755,7 @@ func (c *ClientStartConfig) Factory() (*clientcmd.Factory, error) {
 }
 
 // Clients returns clients for OpenShift and Kube
-func (c *ClientStartConfig) Clients() (*client.Client, *kclient.Client, error) {
+func (c *ClientStartConfig) Clients() (client.Interface, *kclient.Client, error) {
```

Comment:
I agree. But I just changed the minimum necessary to work. If you prefer I can refactor it too.